### DISCUSSION TITLE
About Security /2FA chat.

### DISCUSSION THREAD

**Discussion by zxmd393** [VueTorrent/VueTorrent#1894] [General]
I want to know if there is a security authentication method similar to 2fa on the initial webui or vue, or what methods can be used to perform such security operations.

Thanks all！

**Comment by Larsluph** [comment]
Hi, thanks for your interest about VueTorrent.

Unfortunately qBittorrent only supports username/password combo authentication.
If you want/need more secure authentication, you'll have to use external tools.

**Reply by zxmd393** [reply]
Thank you for your attention, I would like to ask what are the software that can add 2FA to the web interface? Thank you for your reply.

**Reply by Larsluph** [reply]
I personally use the "Traefik" reverse proxy on top of my services to implement basic authentication (username/password).

If you want more advanced protocols, you can also integrate tools like "Authentik" along with Traefik to the auth flow to allow for homemade SSO with 2FA and a lot more instead of the basic auth.

This is the solution I currently have but keep in mind that there are plenty of tools available out there.

**Reply by zxmd393** [reply]
Thank you, I will try it, and if there is a better solution, I will share it. I'll try it first.
